haystack/pipelines/utils.py

```diff
@@ -297,18 +305,23 @@ def _format_wrong_examples_report(
         "document_id", "context", "document_id_and_context", "document_id_or_context", "answer", "document_id_or_answer"
     ] = "document_id_or_answer",
     answer_scope: Literal["any", "context", "document_id", "document_id_and_context"] = "any",
+    wrong_examples_fields: List[str] = None,
+    max_chars: int = None,
 ):
+    fields = ["answer", "context", "document_id"] if wrong_examples_fields is None else wrong_examples_fields
```

Comment:
I'd suggest to make these ones the default value of `wrong_exmaples_fields` wherever it occurs. Otherwise people don't really know what to pass. The default value acts like a suggestion, from which they can drop items or add some they know that should be there.

Reply:
Yeah sure @tstadel ,  what do we do if user intentionally passes None? We default to all available fields or `["answer", "context", "document_id"] `?

Reply:
As we don't make this param optional, None would be invalid. We could handle that for convenience and make an empty list out of it.

Reply:
We can't prevent users from intentionally/accidentally passing None! We currently print all fields in such a case. If we need to change that - lmk.